packages/big-design/src/components/Button/__snapshots__/spec.tsx.snap

```diff
@@ -86,7 +86,7 @@ exports[`isLoading render loading button 1`] = `
   line-height: 2rem;
   outline: none;
   padding: 0 1rem;
-  pointer-events: none;
+  pointer-events: auto;
```

Comment:
I don't really understand what made this change 🤔

Reply:
Ah, looks like `StyledButton` wants the `isLoading` prop passed.
https://github.com/bigcommerce/big-design/blob/master/packages/big-design/src/components/Button/styled.tsx#L32

Reply:
I removed the pointer-events. Repurposed the last commit.